react-native-sdk/update_dependencies.js

```diff
@@ -0,0 +1,35 @@
+const fs = require('fs');
+
+const packageJSON = require('../../YOUR_APP/package.json');
+
+const RNSDKpackageJSON = require('./package.json');
+
+/**
+ * Updates dependencies from the app package.json with the peer dependencies of the RNSDK package.json.
+ */
+function updateDependencies() {
+  let sortedDeps;
+
+  for (const key in RNSDKpackageJSON.peerDependencies) {
+    if (!packageJSON.dependencies.hasOwnProperty(key)) {
+      packageJSON.dependencies[key] = RNSDKpackageJSON.peerDependencies[key];
+      packageJSON.dependencies[key] = '*';
+    }
+
+    sortedDeps = Object.keys(packageJSON.dependencies)
+      .sort()
+      .reduce((item, key) => {
+        item[key] = packageJSON.dependencies[key];
+
+        return item;
+      }, {});
+
+    packageJSON.dependencies = sortedDeps;
+  }
+
+  const data = JSON.stringify(packageJSON, null, 2);
+
+  fs.writeFileSync('package.json', data);
```

Comment:
This path will be incorrect, it needs be in `.../../package.json`. Let's also log that we saved the file and that the user should run npm install again. Of course, that means we need to keep track of the changes we make. If we didn't need to touch the file then we shouldn't save it, and then the cycle stops. Also suggest running pod install the second npm install.